lldb/tools/darwin-debug/CMakeLists.txt

```diff
@@ -1,3 +1,11 @@
+
+# Create an LC_SEGMENT with the special name ExecExtraSuspend which
+# debugserver can detect - it tells debugserver that it will exec a
+# process and that process will start suspended, so debugserver will
+# need to double-resume it to make it run.  A random file is copied
+# into the segment.
+set(CMAKE_EXE_LINKER_FLAGS "${CMAKE_EXE_LINKER_FLAGS} -Wl,-sectcreate,ExecExtraSuspend,ExecExtraSuspend,${CMAKE_CURRENT_SOURCE_DIR}/CMakeLists.txt")
```

Comment:
For some reason, I really dislike putting the CMakeLists.txt file in this section. Wouldn't /dev/null work?

Reply:
@fredriss huh yeah `/dev/null` does work, you end up with a zero length segment/section, whereas putting any content in there eats 4096 bytes to maintain alignment for the next segment.  I'll change to that on the monorepo.